go.mod

```diff
@@ -6,6 +6,10 @@ replace (
 	// Use nested module.
 	github.com/talos-systems/talos/pkg/machinery => ./pkg/machinery
 
+	// See https://github.com/vmware-tanzu/sonobuoy/issues/1520
+	// Remove when v0.55.1+ is released.
+	github.com/vmware-tanzu/sonobuoy v0.55.0 => github.com/vmware-tanzu/sonobuoy v1.11.5-prerelease.1.0.20211111171810-0e7f9c9059d6
```

Comment:
Yeah, that's the right commit: https://github.com/vmware-tanzu/sonobuoy/commit/0e7f9c9059d6